hpx/runtime/naming/name.hpp

```diff
@@ -18,7 +18,7 @@
 #include <hpx/futures/traits/promise_local_result.hpp>
 #include <hpx/modules/itt_notify.hpp>
 #include <hpx/runtime/naming/id_type.hpp>
-#include <hpx/runtime/naming_fwd.hpp>
+#include <hpx/modules/naming.hpp>
 #include <hpx/serialization/serialization_fwd.hpp>
 #include <hpx/serialization/traits/is_bitwise_serializable.hpp>
 #include <hpx/thread_support/atomic_count.hpp>
```

Comment:
Can't comment on the lines further down, but should or could part or all of this file and `name.cpp` be in the `naming` module?

Reply:
Yes, that's the plan. First I need to move the basic types to naming_base, first.